yb-voyager/cmd/importData.go

```diff
@@ -54,6 +54,7 @@ var batchImportPool *pool.Pool
 var tablesProgressMetadata map[string]*utils.TableProgressMetadata
 var importerRole string
 var identityColumnsMetaDBKey string
+var isImportLiveMigrationInSnapshot bool
```

Comment:
Similarly, let's have vars in MSR (IMPORT_DATA_PHASE, IMPORT_DATA_TO_SR_PHASE, IMPORT_DATA_TO_SOURCE_PHASE) ?